hack/windows/Prepare-Node.ps1

```diff
@@ -125,7 +125,7 @@ if ($InstallKubeProxy) {
 
 $StartKubeletFileContent += [Environment]::NewLine + '$global:KubeletArgs += "--cert-dir=$env:SYSTEMDRIVE\var\lib\kubelet\pki --config=/var/lib/kubelet/config.yaml --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf --hostname-override=$(hostname) --pod-infra-container-image=`"mcr.microsoft.com/oss/kubernetes/pause:1.4.1`" --enable-debugging-handlers --cgroups-per-qos=false --enforce-node-allocatable=`"`" --resolv-conf=`"`" --node-ip=$env:NODE_IP"'
 
-if ($ContainerRuntime -eq "containerd") {
+if ($ContainerRuntime -eq "containerd" -and [version]($KubernetesVersion -replace '^v') -lt [version]"1.28.0") {
     $StartKubeletFileContent += [Environment]::NewLine + '$global:KubeletArgs += " --feature-gates=WindowsHostProcessContainers=true"'
```

Comment:
Did WindowsHostProcessContainers retire in 1.28?

Reply:
No WindowsHostProcessContainers flag is not available from 1.28 k8s version. 'Invalid flag' error is returned on using this flag from 1.28 onwards. You can refer to this doc
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/

Reply:
WindowsHostProcessContainers support is there till k8 1.27, need to document this change.